istio-control/istio-autoinject/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.global.defaultPodDisruptionBudget.enabled }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: istio-sidecar-injector
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: sidecar-injector
+    release: {{ .Release.Name }}
+    istio: sidecar-injector
+spec:
+{{- if .Values.global.defaultPodDisruptionBudget.enabled }}
```

Comment:
This if is redundant from line 1 I think

Reply:
Done.